dss/error.py

```diff
@@ -81,7 +117,16 @@ def wrapper(*args, **kwargs):
             code = "read_only"
             title = "The DSS is currently read-only"
             stacktrace = ""
-            headers = {'Retry-After': 600}
+            headers = {'Retry-After': '600'}
+
+        # These errors may be returned by the chalice app before reaching here.
```

Comment:
Can this occur for methods decorated with `dss_handler`?

Reply:
I believe so?  Since this was originally caused by `mock`, I'd have to find a different way to check.

I'm guessing it might return a 500 directly, if say, other components were down (e.g. API gateway) or were erroring/backed up for some other reason.

Reply:
I've tested this a bit now and haven't encountered it hitting this portion of the code other than when using `mock`.